sql-plugin/src/main/301until330-all/scala/org/apache/spark/sql/rapids/shims/RapidsFloorCeilUtils.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.rapids.shims
+
+import org.apache.spark.sql.catalyst.expressions.Expression
+import org.apache.spark.sql.rapids.GpuFloorCeil
+import org.apache.spark.sql.types.{DataType, DecimalType, LongType}
+
+object RapidsFloorCeilUtils {
+
+  def getDataType(child: Expression): DataType = {
```

Comment:
nit: can we rename this to outputDataType? It make it much more clear what is happening in my opinion. 

Also I think I would prefer to have the child's data type passed in instead of the child expression itself.  It makes slightly more code common and also I think makes this method more specific.

But these are both nits that I can live without.

Reply:
done. PTAL